packages/instant-apps-components/src/components/instant-apps-header/instant-apps-header.test.ts

```diff
@@ -0,0 +1,72 @@
+import { expect, test, describe } from 'vitest';
+import '../../../dist/components/instant-apps-header.js';
+const header = document.createElement('instant-apps-header');
+header.setAttribute('id', 'instant-apps-header');
+header.setAttribute('background-color', '#151515');
+
+describe('Testing prop changes', () => {
```

Comment:
the use of `testing` and `test` within `describe()` and `test()` seem redundant, let's try to keep it descriptive but also short/simple. applies to other places.